Docs for Enterprise Hub (#1081)
huggingface/hub-docs: docs/hub/audit-logs.md

# Audit Logs

<svg xmlns="http://www.w3.org/2000/svg" fill="none" aria-hidden="true" focusable="false" role="img" width="1em" height="1em" preserveAspectRatio="xMidYMid meet" viewBox="0 0 33 27"><path fill="currentColor" fill-rule="evenodd" d="M13.5.7a8.7 8.7 0 0 0-7.7 5.7L1 20.6c-1 3.1.9 5.7 4.1 5.7h15c3.3 0 6.8-2.6 7.8-5.7l4.6-14.2c1-3.1-.8-5.7-4-5.7h-15Zm1.1 5.7L9.8 20.3h9.8l1-3.1h-5.8l.8-2.5h4.8l1.1-3h-4.8l.8-2.3H23l1-3h-9.5Z" clip-rule="evenodd"></path></svg>

<Tip warning={true}>

> This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.

Reviewer: ```suggestion
This feature is part of <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
```

</Tip>

Audit Logs enable organization admins to easily review actions taken by members, including organization membership, repository settings and billing changes.

> The resulting view is populated and accessible through your organization admin settings.

Reviewer: ```suggestion
Audit Logs are accessible through your organization admin settings.
```
